ibm/service/iamidentity/data_source_ibm_iam_trusted_profile_identity.go

```diff
@@ -0,0 +1,113 @@
+// Copyright IBM Corp. 2023 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package iamidentity
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM/platform-services-go-sdk/iamidentityv1"
+)
+
+func DataSourceIBMIamTrustedProfileIdentity() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIamTrustedProfileIdentityRead,
+
+		Schema: map[string]*schema.Schema{
+			"profile_id": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ID of the trusted profile.",
+			},
+			"identity_type": &schema.Schema{
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Type of the identity.",
+			},
+			"identifier_id": &schema.Schema{
```

Comment:
can we have common naming convention for "identifier_id" . we are refeering "identifier_id" and "identifier" in different places

Reply:
identifier_id and identifier are two different variables. they are being refered according to the context.